### PR TITLE
[13.0][ADD] purchase_requisition_grouped_by_procurement: This module allows you to group procurements in the same purchase requisitions according to procurement group.

### PR DESCRIPTION
**Pull request by victoralmau**
This module allows you group by purchase requisitions according to procurement.

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT33877

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The module name should be `purchase_requisition_grouped_by_procurement`

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

File: purchase_requisition_grouped_by_procurement/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_requisition_grouped_by_procurement
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/142/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you grouped by purchase requisitions according to procurement.
```
Comment (pedrobaeza) [inline purchase_requisition_grouped_by_procurement/README.rst]:
```suggestion
This module allows you to group procurements in the same purchase requisitions according to procurement group.
```

File: purchase_requisition_grouped_by_procurement/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Purchase > Products > Product**
+#. Create a storable product and set "Procurement" to "Propose a call for tenders" ("Purchase" tab) and set "Routes" to "MTO" + "Buy" ("Inventory" tab).
+#. Create another product the same as the previous one.
```
Comment (pedrobaeza) [inline purchase_requisition_grouped_by_procurement/README.rst:42]: Put this in the configuration.

**Review by pedrobaeza — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**